packages/svelte/src/lib/component/UploadDropzone.svelte

```diff
@@ -0,0 +1,128 @@
+<script lang="ts" generics="TRouter extends FileRouter">
+  import type { FileWithPath } from "file-selector";
+
+  import { classNames, generateClientDropzoneAccept } from "uploadthing/client";
+  import type { FileRouter } from "uploadthing/server";
+
+  import { INTERNAL_uploadthingHookGen } from "../useUploadThing";
+  import Dropzone from "./Dropzone.svelte";
+  import {
+    allowedContentTextLabelGenerator,
+    generatePermittedFileTypes,
+    progressHeights,
+    type UploadthingComponentProps,
```

Comment:
Still looks like this is not separated :/

Reply:
I'm not sure you're looking at the latest changes though 😅

![Captura de pantalla 2023-07-27 124708](https://github.com/pingdotgg/uploadthing/assets/20191823/e66bf218-3a11-4895-954d-75f2aea0b761)

Reply:
Huh, I guess Github is diffing incorrectly 😭